sdk/loadtestservice/load-testing-rest/README.md

```diff
@@ -0,0 +1,192 @@
+# Azure Developer LoadTesting client library for JavaScript
+
+Azure Load Testing provides client library in JavaScript to the user by which they can interact natively with Azure Load Testing service. Azure Load Testing is a fully managed load-testing service that enables you to generate high-scale load. The service simulates traffic for your applications, regardless of where they're hosted. Developers, testers, and quality assurance (QA) engineers can use it to optimize application performance, scalability, or capacity.
+
+**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+
+## Documentation
+
+Various documentation is available to help you get started
+
+<!-- - [Source code][source_code] -->
+- [API reference documentation][api_reference_doc]
+- [Product Documentation][product_documentation]
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 14.x.x or higher
+
+### Prerequisites
+
+- You must have an [Azure subscription](https://azure.microsoft.com/free/) and [Azure Load Test Service Resource](https://learn.microsoft.com/en-us/azure/load-testing/) to use this package.
+
+### Install the `@azure-rest/load-testing` package
+
+Install the AzureLoadTesting client REST client library for JavaScript with `npm`:
+
+```bash
+npm install @azure-rest/load-testing
+```
+
+### Create and authenticate a `AzureLoadTesting`
```

Comment:
Create and authenticate a `AzureLoadTesting` client

Reply:
Updated